tools/check-capitalization

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+from __future__ import absolute_import
+from __future__ import print_function
+
+# check for the venv
+from lib import sanity_check
+sanity_check.check_venv(__file__)
+
+from typing import List, Text
+import json
+import os
+import re
+import subprocess
+import sys
+
+from bs4 import BeautifulSoup
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from scripts.lib.zulip_tools import WARNING, FAIL, ENDC
+
+args = sys.argv[1:]
+subprocess.call(['./manage.py', 'makemessages'] + args,
+                stderr=subprocess.STDOUT)
+
+DJANGO_PO_REGEX = re.compile('msgid "(.*?)"')
+
+CHECK_REGEXES = [re.compile(regex) for regex in [
+    '^[a-z]',
```

Comment:
Can you rename the variable here to something like `DISALLOWED_REGEXES` and add a comment describing the things you're filtering for?